example/lib/custom_code_box.dart

```diff
@@ -108,29 +113,33 @@ class _CustomCodeBoxState extends State<CustomCodeBox> {
       color: Colors.deepPurple[900],
       child: Row(
         children: [
-          Spacer(flex: 2),
-          Text(
+          const Spacer(flex: 2),
+          const Text(
             'Code editor',
             style: TextStyle(fontSize: 28, color: Colors.white),
           ),
-          Spacer(flex: 35),
+          const Spacer(flex: 35),
           codeDropdown,
-          Spacer(),
+          const Spacer(),
           themeDropdown,
-          Spacer(),
+          const Spacer(),
           resetButton,
         ],
       ),
     );
+
     final codeField = InnerField(
```

Comment:
Isn't declaring variables in the build method a bad practice? 

https://stackoverflow.com/questions/71095761/declare-variables-in-build-method-or-in-class-itself-flutter

Reply:
This whole example app is one bad practice.